src/SDKs/_metadata/cognitiveservices_data-plane_AnomalyDetector.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
+Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/moreOver0/azure-rest-api-specs/blob/zhuxia/ad/specification/cognitiveservices/data-plane/AnomalyDetector/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\src\azure-sdk-for-net\src\SDKs
```

Comment:
Was this supposed to be added?

Reply:
Do you mean this line `cmd.exe /c autorest.cmd https://github.com/moreOver0/azure-rest-api-specs/blob/zhuxia/ad/specification/cognitiveservices/data-plane/AnomalyDetector/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\src\azure-sdk-for-net\src\SDKs` ?

I generated the C# code from my own branch because my swagger PR is not merged now. I will update this after the swagger PR get merged.

Reply:
Sorry -- I was asking about the entire text file.

Reply:
I thought I should add this file. It is a auto generated log file when I use generate.ps1 to generate code. I believe this file is target to record repo, branch and commit id.

@dsgouda What do you think about this file? Should I add this file?

Reply:
@AlexGhiondea This is a log file we generate when generating the SDK using scripts. Helps us review what specs were used for it.